src/components/BooksForm.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+
+const mapDispatchToProps = dispatch => ({
+  createBook: book => {
```

Comment:
- [ ] Please move BookForm to src/containers. To that effect kindly create a `containers` folder under the `src` directory.

Reply:
Done.